packages/fds-components/src/style/interactive/toast.css

```diff
@@ -0,0 +1,91 @@
+/**
+* Toast
+*/
+:root {
+  --toast-border-radius: 4px;
+  --toast-maxWidth: 600px;
+}
+
+.toast {
```

Comment:
Should we import the styles for these?

Reply:
All of these styles are imported to a single stylesheet, `src/style/index.css`. 

This is because all the CSS from `fds-components` and `fds-styles` is global by design and meant to be included as a cachable external stylesheet

Reply:
ah ok cool!